### ISSUE TITLE
error on controller installation : update master script deletes iso

### ISSUE THREAD

**Issue by themisa** [clustervision/trinity#383]
**Version:** xcat_mariadb 58fd410
**Environment:** tmaster

```
[root@controller() ~]# grep -i "error" /var/log/xcat/xcat.log 
Error: copycds needs at least one full path to ISO currently.
```

**Comment by hansthen** [comment]
This is an error in the environment. Master needs to have a copy of the installation medium under /trinity/iso

**Comment by themisa** [comment]
Also present in amaster:
**Version:** r5_fixes dbdf5d9
[root@controller trinity]# grep -i error /var/log/xcat/xcat.log 
Error: copycds needs at least one full path to ISO currently.

**Comment by themisa** [comment]
This happened because i used the script update master
```
[root@master ~]# cd /trinity/
[root@master trinity]# ls
clustervision  controller  version
```

**Comment by themisa** [comment]
Fixed in r5_fixes 28de846
